Directory.Build.props

```diff
@@ -1,4 +1,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<ItemGroup>
+  <PackageReference Include="RichCodeNav.EnvVarDump" Version="0.1.1407-alpha" PrivateAssets="all" />
```

Comment:
This should be in eng\target\Settings.props already because of my earlier PR, was something not working?

Reply:
Ah I didn't know we had that change already, thanks! I will remove this after verifying it working